# FormationPositionContoller.msg

float64 count_number
float64 leaderHeading
float64 leaderCommand_yaw
float64 leaderAirspeed
